roadmap.md

```diff
@@ -0,0 +1,38 @@
+# Usage of Epics
+An Epic is a grouping of smaller related issues/tasks that fits into a larger goal. An Epic can potentially span across multiple iterations. 
+
+To track how smaller issues fit into the larger goal, we use [task lists](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-task-lists).
+Breaking a large issue into smaller issues makes the work more manageable and enables team members to work in parallel. 
+It also leads to smaller pull requests, which are easier to review.
+
+>NOTE: To create an epic on Github, use the available Epic template.
+
+# Usage of Github issue template/forms
+We have [customized templates](https://github.com/bf2fc6cc711aee1a0c2a/ffm-project/blob/main/.github/ISSUE_TEMPLATE/epic.md) that helps contributors in 
+creating new issues in the repository. When creating an issue, choose the template that best fits the issue you are about to create.  
+
+# Communicate
+- Use `@mentions` to alert a person or entire team about a the issue. 
+- Use appropriate labels during the creation or the triaging of the issue. Labels help to categorise issues thus improving their discoverbility. Check out the existing labels on - https://github.com/bf2fc6cc711aee1a0c2a/ffm-project/labels.  
+- Assign collaborators to issues to communicate responsibility.
+
+>NOTE: You can always reach out to the wider team via the available [communication channels](https://github.com/bf2fc6cc711aee1a0c2a/ffm-project/wiki/How-to-communicate-in-the-project) 
+
+# View the Roadmap
+The roadmap is defined in the [project view](https://github.com/orgs/bf2fc6cc711aee1a0c2a/projects/5) 
+We have two main views:
```

Comment:
Do the current views make sense? I feel like we should clean these views up in the project and have clear goals on their purpose. 

Please feel free to edit the existing views and remove ones that don't make sense.

Reply:
I didn't change the current views. I wanted to see if there is agreement in having only two views. My reasoning are explained in https://github.com/bf2fc6cc711aee1a0c2a/ffm-project/pull/12#discussion_r811855613

Once we have an agreement, we'll proceed with the cleaning.

Reply:
I missed that, what you have identified makes perfect sense to me